packages/cashier/src/types/shared/websocket.types.ts

```diff
@@ -29,6 +30,8 @@ type TWebSocketCall = {
 
 export type TWebSocket = {
     authorized: TWebSocketCall;
+    cryptoConfig: () => { crypto_config: CryptoConfig };
+    cryptoWithdraw: (args: Omit<CashierInformationRequest, 'cashier' | 'provider' | 'type'>) => Promise<any>;
```

Comment:
```suggestion
    cryptoWithdraw: (args: Omit<CashierInformationRequest, 'cashier' | 'provider' | 'type'>) => Promise<CashierInformationResponse>;
```
@vinu-deriv Any reason for using `any`? 🤔

Reply:
fixed